BSP/Simulator/Src/BSP_SPI.c

```diff
@@ -6,18 +6,20 @@
 #include "BSP_SPI.h"
 #include "config.h"
 #include "common.h"
+#include <inttypes.h>
+#include "Simulator.h"
 
-void SPI3_Init(bsp_os_t *spi3_os){
-	// TODO
-}
+static bool initialized = false;
 
 /**
  * @brief   Initializes the SPI port.
  * @param   port The SPI port to initialize.
  * @return  None
  */
 void BSP_SPI_Init(spi_port_t port, bsp_os_t *spi_os, bool baremetal) {
-    // TODO
+    initialized = true;
```

Comment:
Do we actually use SPI in the simulator? I thought we bypassed it everywhere

Reply:
I just added some basic stuff since I was debugging it when debugging the LTC2315. Not really necessary, but doesn't hurt in case we do stuff with it later.

Reply:
Ok. We just need to be aware that not all of the SPI traffic that there would be in the real system will show up in the simulator

Reply:
I will add a comment that says that.